### PR TITLE
Added resource PerfectBugReport.io as general web dev resource

### PR DESCRIPTION
**Pull request by vandernath**
## Resource name and link

[PerfectBugReport.io](https://www.perfectbugreport.io/)

## Short description

A simple checklist of the essential items to include in bug reports.

## Why do you think this is a good resource?

We spend too much time trying to decipher bug reports from non-technical people. Hopefully, spreading this resource around—as an educational tool for people to learn how to report bugs properly—will help!

## GitHub repository (optional)

N/A

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -272,4 +272,5 @@ This is a list showing the GitHub usernames of all who have contributed to this
 - [@serhatbek](https://github.com/serhatbek)
 - [@ChrRepou](https://github.com/ChrRepou)
 - [@web-roberto](https://github.com/web-roberto)
-- [@Bandit-Army](https://github.com/Bandit-Army)
\ No newline at end of file
+- [@Bandit-Army](https://github.com/Bandit-Army)
+- [@vandernath] (https://github.com/vandernath)
```
Comment (sophiabrandt) [inline CONTRIBUTORS.md:276]: Please remove the empty space between the braces
```suggestion
- [@vandernath](https://github.com/vandernath)
```

File: generalResources.md [hunk 2]
```diff
@@ -52,4 +52,6 @@
 
 * [**CompSciLib**](https://www.compscilib.com/): A free site for learning and practicing Computer Science theory topics in courses such as Computer Organization, Discrete Math, Networking, etc. Think Khan Academy, but for Computer Science.
 
-* [**frontend Mentor**](https://www.frontendmentor.io/challenges): Solve real-world HTML, CSS and JavaScript challenges whilst working to professional designs.
\ No newline at end of file
+* [**frontend Mentor**](https://www.frontendmentor.io/challenges): Solve real-world HTML, CSS and JavaScript challenges whilst working to professional designs.
+
+* [**PerfectBugReport.io**] (https://www.perfectbugreport.io/): A simple checklist of the essential items to include in bug reports.
```
Comment (sophiabrandt) [inline generalResources.md:57]:
```suggestion
* [**PerfectBugReport.io**](https://www.perfectbugreport.io/): A simple checklist of the essential items to include in bug reports.
```
Please remove the empty space between the braces

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
See suggestions

**Comment by vandernath** [comment]
Thanks for reviewing @sophiabrandt , I hope this works!

**Pull request merged**